packages/client/components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
```

Comment:
-1 can you move it to `client/ui/Dialog`? I'd love to separate, generic components from stuff related to our features. I did that here: [feat: added button component #8264](https://github.com/ParabolInc/parabol/pull/8264) so it'd be consistent